packages/synapse-react-client/src/components/download_list/AddToDownloadListConfirmationAlert/AddToDownloadListConfirmationAlert.tsx

```diff
@@ -0,0 +1,76 @@
+import { displayToast } from '@/components/index'
+import { ErrorBanner } from '@/components/error/ErrorBanner'
+import {
+  useAddToDownloadList,
+  useGetAddToDownloadListStats,
+} from '@/synapse-queries/download/useDownloadList'
+import { useSynapseContext } from '@/utils'
+import { AddToDownloadListRequest } from '@sage-bionetworks/synapse-client'
+import DownloadConfirmationUI from '../DownloadConfirmationUI'
+import { displayFilesWereAddedToDownloadListSuccess } from '../DownloadConfirmationUtils'
+
+export type AddToDownloadListConfirmationAlertProps = {
+  /**
+   * The "AddToDownloadListRequest" that would be used to add items to the download list.
+   */
+  addToDownloadListRequest: AddToDownloadListRequest
+  /**
+   * Invoked when the dialog is closed, either by cancelling or successfully adding to the download list.
+   */
+  onClose: () => void
+}
+
+/**
+ * A component that displays a confirmation alert when adding multiple items to the download list though a container or
+ * query.
+ * @param props
+ * @constructor
+ */
+function AddToDownloadListConfirmationAlert(
```

Comment:
We now have a generic API that handles 
* projects/folders
* table queries
* datasets & dataset collections

This new component will use the generic API. We will remove/deprecate the entity-type specific components that do the same thing.